README.md

```diff
@@ -7,35 +7,38 @@
 </h1>
 
 <p align="center">
-	Quotes Status Creator lets you share quotations as status images on social media
+	Share quotes as status images on social media
```

Comment:
I think `Quotes Status Creator lets you share quotations as status images on social media` was better. It's the short description for our app on IzzyOnDroid, PlayStore etc What do you think?

Reply:
I agree, and here in the README it says "Quotes Status Creator" right above it, so it isn't needed here.